src/workflow/show_mnemonic.c

```diff
@@ -33,8 +33,8 @@
 
 #define NUM_RANDOM_WORDS 5
 
-static const char* _back_label = "Back to seed phrase";
-static const char* _cancel_confirm_title = "Mnemonic";
+static const char* _back_label = "Back to\nrecovery words";
+static const char* _cancel_confirm_title = "Recovery\nwords";
```

Comment:
This could still fit on one line though, diesn't it?

Reply:
No, it is a bit too close to the `x` and checkmark in the title.